src/main/java/org/jabref/logic/formatter/bibtexfields/NormalizeMonthFormatter.java

```diff
@@ -23,6 +26,31 @@ public String getKey() {
     public String format(String value) {
         Objects.requireNonNull(value);
         Optional<Month> month = Month.parse(value);
+
```

Comment:
Since this is about parsing months, it should go into the `Month.parse` method, right?

Reply:
Moved it there